.auri/$m5bch37h.md

```diff
@@ -0,0 +1,6 @@
+---
+package: "@lucia-auth/adapter-prisma" # package name
+type: "patch" # "major", "minor", "patch"
+
+adjust peerDependency to solve the unmet peer dependency warning for prisma 5.x
```

Comment:
Since this is frontmatter, move the “Adjust peerDependecy…” outside the —-. Capitalize “adjust” as well.